### DISCUSSION TITLE
How can we clear error bag from alphine?

### DISCUSSION THREAD

**Discussion by abrardev99** [livewire/livewire#4008] [1. Help] [answered]
I have a Livewire component and using Alpine with it. I want to clear the validation error bag from Alpine. I can not access `$wire.resetValidation `method in Alpine.

**Comment by joshhanley** [comment, marked as answer]
@abrardev99 I believe only methods within your component can be called from the front end, but none of the built in methods.

You might be able to work around it by creating a method in your component called `resetValidation` and just call the parent method in that. But I haven't tested if that works or not.

```php
public function resetValidation($field = null)
{
    parent::resetValidation($field);
}
```

Hope this helps!

**Reply by abrardev99** [reply]
This actually works and I'm already using this approach. Wondering if we can call built in methods, which I guess is not possible ?

**Reply by joshhanley** [reply]
@abrardev99 nah it's a security concern if all methods were just freely available from the front end, so you can't call the built in methods, unless you override them in your component class.
